docs/conf.py

```diff
@@ -8,7 +8,7 @@
 import os
 import sys
 
-from src import eda_analysis
+import eda_analysis
```

Comment:
Do we need to move the line `sys.path.insert(0, os.path.abspath('..'))` before this import? I need to move it locally to run `poetry run make html`

Reply:
```suggestion
sys.path.insert(0, os.path.abspath('..'))
import eda_analysis
```

Reply:
I guess; we can always change it back if that doesn't work.